ansible_collections/arista/avd/plugins/action/anta_catalog_builder.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from typing import Any, Dict, List, Tuple
+
+from ansible.plugins.action import ActionBase
+from anta.loader import parse_catalog
+
+TestTuple = Tuple[str, Dict[str, str]]
+Catalog = Dict[str, List[Dict[str, Any]]]
+
+
+AVD_MAPPING: Dict[str, List[TestTuple]] = {
+    "management_api_http": [("anta.tests.security.VerifyAPIHttpStatus", {}), ("anta.tests.security.VerifyAPIHttpsSSL", {"profile": "https_ssl_profile"})]
+}
+
+
+def _get_package_name(test_name: str) -> str:
+    """Extracts the package name from a fully qualified test name."""
+    return test_name.rsplit(".", 1)[0]
+
+
+def _get_test_name(test_name: str) -> str:
+    """Extracts the test name from a fully qualified test name."""
+    return test_name.rsplit(".", 1)[1]
+
+
+def _update_test_parameters(value: Any, test_parameters: Dict[str, str]) -> Dict[str, Any]:
+    """Updates the test parameters based on the provided structured configuration values."""
+    return {param_key: value.get(structured_config_key) for param_key, structured_config_key in test_parameters.items()}
```

Comment:
Maybe you could use our `get()` util to support deeper nested keys? Check it out in `plugins/plugin_utils/utils/get.py`

Reply:
Nice little tool! I'm now using it in ANTA :)